cmd/testdata/fixtures/workspace-generate/packages/button/demo/basic.html

```
@@ -0,0 +1 @@
+<test-button label="Hello"></test-button>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)